chef/data_bags/crowbar/template-network.json

```diff
@@ -87,6 +87,26 @@
             },
             "intf2": {
               "if_list": [ "1g1", "1g2" ]
+            },
+            "intf3": {
+              "if_list": [ "1g1", "1g2" ]
+            }
+          }
+        },
+        {
+          "pattern": "quad/.*/.*",
```

Comment:
The idea is that many/most servers have four network interfaces and with this mode, default Crowbar mechanism will distribute the networks on those interfaces. This way intf0-3 is actually mapped to physical interfaces. It can be changed by hand in the network.json but with this mode it would be only one parameter change with pretty sane default behavior.
For Ironic this is pretty important as it needs it's own physical network to handle the network booting of baremetal hosts.